tests/Fixer/Strict/DeclareStrictTypesFixerTest.php

```diff
@@ -141,11 +155,11 @@ public function provideMessyWhitespacesCases()
     {
         return [
             [
-                "<?php declare(strict_types=1);\r\nphpinfo();",
+                "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
```

Comment:
> do not change existing tests that are verifying if the feature works.
> only add new tests.

This test needs changing either way.
1. If we want to match (proposed) PSR-12 by default, we need the add the newline by default.
2. The feature and its tests are broken in release version, the full test case is
   ```php
   [
       "<?php\r\ndeclare(strict_types=1);\r\n\tphpinfo();",
       "<?php\r\n\tphpinfo();",
   ],
   ```
   The `\t` should have never been removed.